v03_pipeline/lib/reference_datasets/reference_dataset.py

```diff
@@ -73,7 +74,7 @@ def get_ht(
         )
         path = self.raw_dataset_path(reference_genome)
         ht = module.get_ht(path, reference_genome)
-        if self.enums:
+        if self.enums and not CONFIG[self].get(SKIP_ENUM_ANNOTATION):
```

Comment:
I don't love the added config option for this; I think it's confusing that we use it here and then three lines below call `annotate_globals` without referencing the option.  

What if, instead, we modify `get_enum_select_fields` to not hard fail if the field to enum map isn't present on the table?

Reply:
ooh I like that!